pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -143,11 +143,7 @@ public ProxyServiceStarter(String[] args) throws Exception {
             // load config file
             config = PulsarConfigurationLoader.create(configFile, ProxyConfiguration.class);
 
-            if (isBlank(metadataStoreUrl)) {
```

Comment:
_Use zookeeperServers from command line if metadataStoreUrl is empty._  How to handle if the metadataStoreUrl is empty ?

Reply:
getMetadataStoreUrl() uses zookeeperServers by default,